src/phoneField/index.tsx

```diff
@@ -75,10 +77,10 @@ const mapCountriesPhoneData = (
   countriesName: customCountriesName,
 ): mappedCountriesPhoneData[] => {
   return countriesData.map(countryData => ({
-    value: `+${countryData.dialCode}`,
+    value: countryData.dialCode,
```

Comment:
Could we rename value (which has not intrinsic meaning) to something more semantic to help readability? maybe dialCode or isdCode ?

Reply:
I had to keep the object format {value: '', label: ''} to match with the SelectField API component to build options.